graylog2-server/src/main/java/org/graylog2/indexer/cluster/Cluster.java

```diff
@@ -155,9 +165,16 @@ public boolean isConnected() {
                 .timeout(Ints.saturatedCast(requestTimeout.toSeconds()))
                 .build();
 
-        final JestResult result = JestUtils.execute(jestClient, request, () -> "Couldn't check connection status of Elasticsearch");
-        final int numberOfDataNodes = result.getJsonObject().path("number_of_data_nodes").asInt();
-        return numberOfDataNodes > 0;
+        try {
+            final JestResult result = JestUtils.execute(jestClient, request, () -> "Couldn't check connection status of Elasticsearch");
+            final int numberOfDataNodes = result.getJsonObject().path("number_of_data_nodes").asInt();
+            return numberOfDataNodes > 0;
+        } catch (ElasticsearchException e) {
+            if (LOG.isDebugEnabled()) {
+                LOG.error(e.getMessage(), e);
```

Comment:
This was intentional. The idea was to only show the error if debugging is enabled so it's easier to identify the log message is an error.